packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-configs.tsx

```diff
@@ -0,0 +1,416 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import {
+    AUTH_STATUS_CODES,
+    idv_error_statuses,
+    isNavigationFromDerivGO,
+    isNavigationFromP2P,
+    routes,
+} from '@deriv/shared';
+import VerificationStatusActionButton from '../../../Components/verification-status-action-button';
+import { TAuthStatusCode } from '../../../Types/common.type';
+import { TIDVErrorStatus } from '../../../Helpers/utils';
+
+export const submission_status_code = {
+    selecting: 'selecting',
+    submitting: 'submitting',
+    complete: 'complete',
+} as const;
+
+export const service_code = {
+    idv: 'idv',
+    onfido: 'onfido',
+    manual: 'manual',
+} as const;
+
+type TAuthStatus = {
+    needs_poa?: boolean;
+};
+
+export const getPOIStatusMessages = (
+    status: TAuthStatusCode,
+    auth_status?: TAuthStatus,
+    should_show_redirect_btn?: boolean,
+    is_from_external?: boolean
+) => {
+    const resubmitButton = (onClick?: () => void, platform_name?: string) => (
+        <React.Fragment>
+            <VerificationStatusActionButton
+                button_text={<Localize i18n_default_text='Upload Document' />}
+                onClick={onClick}
+            />
+            {!is_from_external && should_show_redirect_btn && (
+                <VerificationStatusActionButton
+                    button_text={<Localize i18n_default_text='Back to {{platform_name}}' values={{ platform_name }} />}
+                    onClick={onClick}
+                />
+            )}
+        </React.Fragment>
+    );
+
+    const verifiedButton = (onClick?: () => void, platform_name?: string) => {
+        if (is_from_external) return null;
+        if (auth_status?.needs_poa) {
+            return (
+                <VerificationStatusActionButton
+                    to={routes.proof_of_address}
+                    button_text={<Localize i18n_default_text='Submit proof of address' />}
+                />
+            );
+        }
+        if (should_show_redirect_btn)
+            return (
+                <VerificationStatusActionButton
+                    button_text={<Localize i18n_default_text='Back to {{platform_name}}' values={{ platform_name }} />}
```

Comment:
platform_name is optional props. Is that okay here?

Reply:
yes, because we don’t need them in some of the scenarios